docs/about/governance.md

```diff
@@ -0,0 +1,59 @@
+# Governance
+
+## Contributors
+
+Anyone involved with Stylelint is considered a contributor.
+
+Contributors:
+
+- report bugs
+- request new features
+- improve documentation
+- fix bugs
+- add new features
+- improve existing code
+
+All proposed changes to Stylelint require consensus to move forward.
+
+## Members
+
+Several active members also support Stylelint.
+
+Members are all volunteers who donate time and effort. They do what they can when they can.
+
+Members:
+
+- triage issues
+- review code on pull requests
+- ensure code quality
+
+To become a member:
+
+- 1 owner must champion an active contributor
+- 2 or more owners other than the champion must agree, and no owners disagree
```

Comment:
Can an existing member propose to the owners for an active contributor to become a member?

Reply:
Yes, anyone can propose for an active contributor to become a member, including that person themselves.

An owner would then volunteer to champion/advocate for that person.